core/types/block.go

```diff
@@ -640,8 +640,43 @@ type RawBody struct {
 	Requests     Requests
 }
 
+// Represents original transaction number in blockchain (do not includes system txs in block)
```

Comment:
1. ` Represents original transaction number in blockchain` - no. by naming (see `Dev.md`): 
- Num - it's blockchain-related number - same on all nodes
- ID - it's our internal auto-increment - not related to blockchain and different on all nodes. 

2. `BaseTxID` let's rename it to `BaseTxnID`. Also let's rename `txID` to `txnID`. (also by naming - see `Dev.md`)